Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/IconOptionsView/IconOptionsViewController.swift

```diff
@@ -0,0 +1,48 @@
+import UIKit
+
+public final class IconOptionsViewController: UIViewController {
```

Comment:
I know in this current implantation of this screen we are using it with the `OptionView` you've made, but because this screen is now so flexible it include anything in the `[UIView]` that I'm not sure including _Option_ in the name is helpful.

I'm not great at naming, so I'm not sure I have the answer to this. But here are some ideas for discussion

`IconScreenViewController` 
`FlexibleIconViewController`
`IconStackViewController`

Reply:
It's a good point, I like `IconScreenViewController` as the most non-specific option here, will opt for that 🙂